types/chart.js/index.d.ts

```diff
@@ -602,6 +602,7 @@ declare namespace Chart {
         maxBarThickness?: number;
         minBarLength?: number;
         scaleLabel?: ScaleTitleOptions;
+        labels?: string[];
```

Comment:
Taking a look to the documentation, I think this property does not belong to that interface. Please provide the link to the documentation or code/live examples that shows this property behavior.

Reply:
https://jsfiddle.net/Shnurilo/ya50msop/

Reply:
Now, please fulfill the PR message as suggested by @lekoaf.